aeron-client/src/main/java/io/aeron/logbuffer/LogBufferDescriptor.java

```diff
@@ -135,11 +135,21 @@ public class LogBufferDescriptor
      */
     public static final int LOG_DEFAULT_FRAME_HEADER_OFFSET;
 
+
     /**
      * Maximum length of a frame header.
      */
     public static final int LOG_DEFAULT_FRAME_HEADER_MAX_LENGTH = CACHE_LINE_LENGTH * 2;
 
+    /**
+     * The maximum byte of the channel-uri stores as an ascii string (excluding the length).
+     */
+    public static final int LOG_CHANNEL_URI_MAX_LENGTH = 4096 ;
```

Comment:
There will be `ChannelUri.MAX_URI_LENGTH` that is set to `4095`, i.e. we don't need log buffer specific limit.

Reply:
4095 or 4096? I guess the latter.

Reply:
I guess for /0 termination of C based strings.